### DISCUSSION TITLE
Readme only update after manually trigger a workflow

### DISCUSSION THREAD

**Discussion by kuanhn** [athul/waka-readme#58] [Q&A] [answered]
Hi athul,

I have set up the GitHub action workflow for a while but my readme only gets updated after I manually trigger it.
These schedule workflows are still running daily and are successful but there are no new commits pushed by the GitHub action.
Can you help me to check my config?

https://github.com/kuanhn/kuanhn/blob/master/.github/workflows/waka.yml

**Comment by kuanhn** [comment, marked as answer]
Oh! it works after the commit https://github.com/kuanhn/kuanhn/commit/412010f6c512fab229a702cf8082964dc5511440

**Reply by athul** [reply]
Hey,
Really sorry for not responding before. Had been stuck with some academic activities. Great to hear that it worked out for you.

**Reply by kuanhn** [reply]
Hi @athul 

Sorry for contacting you during a busy time. There are some updates.
It seems that after the commit above, the only thing the date in the title is is updated.
https://github.com/kuanhn/kuanhn/commits/master
And here is the commit when I trigger workflow by manually.
https://github.com/kuanhn/kuanhn/commit/acab03edabcf87a1ca526c1ad3c8d118e21b3aab

**Reply by jamesmcroft** [reply]
I seem to be suffering from the same issue. The date title is being updated when the CRON job runs, but the data won't. 

I have to manually run the workflow in order to get the data to update.
